public/gameInfo.php

```diff
@@ -1266,8 +1264,11 @@ function ResetProgress() {
                                 $completionPctHardcore = sprintf("%01.2f", ($wonByHardcore / $numDistinctPlayersCasual) * 100);
                             }
 
+                            $imgFadeClasses = 'grayscale blur-[0.8px] contrast-[0.65] hover:grayscale-0 hover:contrast-100 hover:blur-0 transition duration-300';
```

Comment:
This is too much blur.

![image](https://user-images.githubusercontent.com/32680403/233149738-a1922088-e4d1-49ad-aa22-c37571179893.png)

Here's how the `_lock` file is created:
https://github.com/RetroAchievements/RAWeb/blob/47f3e56f4933b89f5f22e990d9e0e21e960339a7/app_legacy/Helpers/util/media.php#L109-L113

Reply:
Also note that the icons for game 16839 were specifically designed to leverage the greyscale algorithm, and don't appear to be greyscaling (or contrasting) exactly the same.

> When possible, I've used cyan and orange(mostly because a chart seemed to indicate that these colors contrast each other even when someone is colorblind) with the exact same shade-Value to create badges that are "invisible" when they are locked, but become visible when unlocked. Hopefully, this will allow people who may have never played MYST before to play mostly-blind without the achievements or badges spoiling what to do or what clues are necessary.

![image](https://user-images.githubusercontent.com/32680403/233151963-a3628a9f-d697-4d8a-8d0c-9cc6c0aedb3c.png)